lib/sidekiq/cron.rb

```diff
@@ -1,8 +1,30 @@
 require "sidekiq/cron/job"
+require "sidekiq/cron/namespace"
 require "sidekiq/cron/poller"
 require "sidekiq/cron/launcher"
 
 module Sidekiq
   module Cron
+    Redis.respond_to?(:exists_returns_integer) && Redis.exists_returns_integer =  false
```

Comment:
this is deprecated and should be removed

Reply:
Sorry about this one, I was confused during the `rebase`.